G/GAP_pkg/GAP_pkg_juliainterface/build_tarballs.jl

```diff
@@ -18,6 +18,8 @@ version = offset_version(upstream_version, offset)
 # Collection of sources required to build this JLL
 sources = [
     GitSource("https://github.com/oscar-system/GAP.jl", "c33287b416ca30fcc386a6b192d67c1955fdae8f"),
+    ArchiveSource("https://julialang-s3.julialang.org/bin/musl/x64/1.6/julia-1.6.7-musl-x86_64.tar.gz",
```

Comment:
@giordano apologies for the stupid question (I feel I should know this, but embarrassingly I don't) but: are such sources cached by the system in some way? so that we don't end up downloading it again and again

Reply:
Yes, all sources are downloaded inside Julia's depot which is persistent, but each runner has a different depot, so between different jobs you may end up downloading the file again, but hopefully not again in the following job on the same runner (unless the depot is cleaned up in the meantime because it reached size limit).